### DISCUSSION TITLE
On Windows (at least) get offset of 0,0 within the virtual screen

### DISCUSSION THREAD

**Discussion by naikrovek** [go-vgo/robotgo#380] [Ideas]
On systems with multiple monitors, the leftmost monitor is not always the primary monitor.  That gives positions on this monitor a negative X value.

The existing `GetScreenSize` function is fine for getting the overall resolution of the virtual screen, but not where its origin is.  I'd like to know where the origin is, as well as the overall screen size.

The Win32 API supports this via the [`GetSystemMetrics`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getsystemmetrics) function when supplied with the `SM_XVIRTUALSCREEN` and `SM_YVIRTUALSCREEN` flags, which get the X location of the left side of the virtual screen and the Y location of the top of the virtual screen, respectively.  From this, the offset of the coordinate origin is easily obtained.

**Comment by vcaesar** [comment]
The Windows multi screen will be support when I have time.
